example/ios/Podfile

```diff
@@ -28,7 +28,7 @@ target 'example' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable the next line.
-  use_flipper!()
```

Comment:
Why this change?

Reply:
Because it significantly sped up the CI, and it's not used that often (if any). Am I right @BartoszKlonowski ?